test/integration/targets/azure_rm_sqldatabase/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+- name: Create SQL Server
+  azure_rm_sqlserver:
+    resource_group: "{{ resource_group }}"
+    name: sql-test-server-dauih
+    location: eastus
+    admin_username: mylogin
+    admin_password: Testpasswordxyz12!
+- name: Create instance of SQL Database
```

Comment:
Are you able to add a check mode and idempotent task for the create, update and delete tasks